client/trino-jdbc/src/test/java/io/trino/jdbc/TestTrinoDriverUri.java

```diff
@@ -104,8 +104,8 @@ public void testInvalidUrls()
         // ssl verification without ssl
         assertInvalid("jdbc:trino://localhost:8080?SSLVerification=FULL", "Connection property SSLVerification requires TLS/SSL to be enabled");
 
-        // ssl verification using port 443 without ssl
-        assertInvalid("jdbc:trino://localhost:443?SSLVerification=FULL", "Connection property SSLVerification requires TLS/SSL to be enabled");
+        // ssl verification using port 8080 without ssl
+        assertInvalid("jdbc:trino://localhost:8080?SSLVerification=FULL", "Connection property SSLVerification requires TLS/SSL to be enabled");
 
         // ssl key store password without path
```

Comment:
why? Seems like use of 443 here was intentional

Reply:
This was an incorrect test